### PR TITLE
Add group leader section

### PR DESCRIPTION
**Pull request by jordanwade**
TO TEST: 
Go to http://localhost:3000/community/sisterhood or http://localhost:3000/community/
The leader section is at the bottom of those pages.

image

### CODE REVIEW

File: components/CommunityBeLeaderSection/CommunityBeLeaderSection.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { Box, Button, Icon } from 'ui-kit';
+
+export default function CommunityBeLeaderSection(props) {
```
Comment (rkdavidson) [inline components/CommunityBeLeaderSection/CommunityBeLeaderSection.js:3]: Quasi-nitpicky, but this component name feels clumsy.

What about something like `CommunityLeaderActions` or `CommunityLeaderSection`?

**Review by rkdavidson — COMMENTED** [review]
Tested locally and looks good to me.

Have some thoughts on responsive for the page as a whole but we're getting there, and beyond scope of this PR I think.

**Review by rkdavidson — APPROVED** [review]

File: components/CommunityBeLeaderSection/CommunityBeLeaderSection.js (hunk 1 above)
Comment (drewbarontini) [reply components/CommunityBeLeaderSection/CommunityBeLeaderSection.js:3]: Agreed. I like `CommunityLeaderActions`.

**Review by drewbarontini — COMMENTED** [review]
One note on that component name, but looking good!

**Review by drewbarontini — DISMISSED** [review]

**Review by drewbarontini — DISMISSED** [review]

**Comment by jordanwade** [comment]
That is a little annoying to have to get another review just for merging main 🤷‍♂️ Thank you @drewbarontini

**Comment by jordanwade** [comment]
Ok, that one was my fault 🤦‍♂️

**Review by drewbarontini — APPROVED** [review]

**Pull request merged**